Specs/GCMV8.lean

```diff
@@ -188,10 +187,8 @@ def GCMInitV8 (H : BitVec 128) : (List (BitVec 128)) :=
   [H0_rev, H1, H2_rev, H3_rev, H4, H5_rev, H6_rev,
    H7, H8_rev, H9_rev, H10, H11_rev]
 
--- set_option profiler true in
--- set_option maxRecDepth 20000 in
--- set_option maxHeartbeats 2000000 in
--- unseal pmod.pmodTR degree.degreeTR reverse.reverseTR pmult.pmultTR Nat.bitwise in
+set_option maxRecDepth 20000 in
```

Comment:
This `maxRecDepth` increase is a bit scary, do we need it? Increasing the maxRecDepth can cause Lean to rather ungracefully crash with stack-overflows

Reply:
I have tried and was able to reduce depth to 8000. I was also able to remove the maxHeartbeat. Unfortunately these maxRecDepth could not be fully removed.

Reply:
I guess it's fine to have them, if it can't be avoided. Better than having `native_decide`, anyway